snapshot/git/gitdb/git_tags.go

```diff
@@ -0,0 +1,92 @@
+package gitdb
+
+import (
+	"fmt"
+	"strings"
+
+	snap "github.com/scootdev/scoot/snapshot"
+)
+
+type TagsConfig struct {
+	Name   string
```

Comment:
Argh, the IDs were too long because we were using Remote instead of Name; thanks; fixed.